app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,61 @@
+class MoviesController < ApplicationController
+  def index
+    @movies = Movie.all.order({ :created_at => :desc })
+
+    render({ :template => "movies/index.html.erb" })
+  end
+
+  def show
+    the_id = params.fetch("the_movie_id")
+    @movie = Movie.where({:id => the_id }).first
```

Comment:
Here we have `.first`; in other places (e.g. in the `update` action) we have `.at(0)`; let's be consistent. How about `.at(0)` everywhere?